packages/enr/src/crypto.ts

```diff
@@ -14,14 +19,11 @@ export async function sign(
   message: Uint8Array,
   privateKey: Uint8Array
 ): Promise<Uint8Array> {
-  const [signature, recoveryId] = await secp.sign(message, privateKey, {
-    recovered: true,
-    der: false
-  });
-  return concat(
-    [signature, new Uint8Array([recoveryId])],
-    signature.length + 1
-  );
+  const signature = nobleSign(message, privateKey);
+  return concat([
+    signature.toCompactRawBytes(),
+    new Uint8Array([signature.recovery || 0])
```

Comment:
Are we sure that an `undefined` `signature.recovery` means the value should be `0`?